### DISCUSSION TITLE
Stop pinned tabs staying in every workspace

### DISCUSSION THREAD

**Discussion by tyler-jurgens-s2d** [Floorp-Projects/Floorp#1398] [Feature Request]
I use workspaces extensively to keep different aspects of my browsing separate. However, when I pin a tab in Floorp it will remain as a pinned tab in every workspace.

This is not ideal because I may have different workspaces for different tasks. For example - I may have a workspace for email and general browsing, so I want to pin my email to the first tab. 

On an additional workspace I may want to be checking forums for a specific vendor I follow. I may wan to pin the main forum on that workspace. I don't need (or want) to see my email pinned tab, nor when I switch back to my email do I want to see the forum tab. 

Thank you for all you do.

**Comment by Grazouillis** [comment]
I do agree. Is it possible to consider this request for an upcoming version please ? :pray:
